doc/classes/Node.xml

```diff
@@ -82,7 +82,13 @@
 			<description>
 				Called when there is an input event. The input event propagates up through the node tree until a node consumes it.
 				It is only called if input processing is enabled, which is done automatically if this method is overridden, and can be toggled with [method set_process_input].
-				To consume the input event and stop it propagating further to other nodes, [method Viewport.set_input_as_handled] can be called.
+				To consume the input event and stop it propagating further to other nodes, [method Viewport.set_input_as_handled] can be called. 
+				[codeblock]
+				func _input(event: InputEvent) -> void:
+					if event.is_action_pressed("action_name"):
+						# excute code
+						get_viewport().set_input_as_handled()
```

Comment:
```suggestion
				func _input(event):
				    if event.is_action_pressed("action_name"):
				        # Excute code.
				        get_viewport().set_input_as_handled()
```
We [don't add](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing) static typing to the documentation 

Also code examples are indented with spaces